Foundation/NSFileManager.swift

```diff
@@ -576,7 +576,7 @@ open class FileManager : NSObject {
     
         If you wish to only receive the URLs and no other attributes, then pass '0' for 'options' and an empty NSArray ('[NSArray array]') for 'keys'. If you wish to have the property caches of the vended URLs pre-populated with a default set of attributes, then pass '0' for 'options' and 'nil' for 'keys'.
      */
-    open func enumerator(at url: URL, includingPropertiesForKeys keys: [URLResourceKey]?, options mask: DirectoryEnumerationOptions = [], errorHandler handler: (@escaping (URL, Error) -> Bool)? = nil) -> DirectoryEnumerator? {
+    open func enumerator(at url: URL, includingPropertiesForKeys keys: [URLResourceKey]?, options mask: DirectoryEnumerationOptions = [], errorHandler handler: ((URL, Error) -> Bool)? = nil) -> DirectoryEnumerator? {
```

Comment:
I've been talking about it with @milseman and @slavapestov. Right now it *is* treated as `@escaping`, so it's not *broken;* a function type used as a generic argument is assumed to be used in arbitrary ways inside the body of the type, and that includes Optional. But we probably want to special-case Optional at some point, and then we'll have to be careful to put this back.